build/Common.props

```diff
@@ -40,7 +40,7 @@
     <StackExchangeRedisPkgVer>[2.1.58,3.0)</StackExchangeRedisPkgVer>
     <StyleCopAnalyzersPkgVer>[1.1.118,2.0)</StyleCopAnalyzersPkgVer>
     <SystemCollectionsImmutablePkgVer>[1.4.0,5.0]</SystemCollectionsImmutablePkgVer>
-    <SystemDiagnosticSourcePkgVer>[5.0.0-rc.2.20475.5]</SystemDiagnosticSourcePkgVer>
+    <SystemDiagnosticSourcePkgVer>[5.0.0]</SystemDiagnosticSourcePkgVer>
```

Comment:
@cijothomas Should it be:

`<SystemDiagnosticSourcePkgVer>[5.0.0,6.0)</SystemDiagnosticSourcePkgVer>`

Also these other dependencies like...

`<SystemTextJsonPkgVer>[4.7.0,5.0)</SystemTextJsonPkgVer>`

Should we also kick them to accept 5.0? Eg:

```xml
<SystemTextJsonPkgVer>[4.7.0,6.0)</SystemTextJsonPkgVer>
```

I'm getting a lot of errors upgrading a solution to .NET 5 because OTel requires < 5.0.0 on a lot of things.

Reply:
Thanks - I made all System packages to be accepted upto 6.